### ISSUE TITLE
Passwordless authentication (aka magic link) - new browser window

### ISSUE THREAD

**Issue by stievendhooge** [damienbod/angular-auth-oidc-client#389]
I'm trying to implement passwordless authentication for an Angular app (implicit flow).

Following has been setup:
- identity server that ask for mail adres as login
- login link is sent to the mail adres
- clicking the link will log the user in with the identity server
- redirect to the app

The library (v9.0.2) is configured via load_using_stsServer and uses the wellKnownEndpoints exposed by the identity server.

The issue:
When clicking the link in the mail, login on the identity server and redirection to the app occurs as expected. However, the token validation process fails so I'm redirected to the unauthorized page of my app.

What I found out so far:
If I paste the magic link in the browser tab that initiated the login procedure, login and redirection works as expected.
If I paste the magic link in a new tab, login and redirection fails.
This made me think my issue is sessionState related. So I changed the storage to use localStorage. This does indeed fix the issue. I can login correctly in a new tab.
However, it only fixes the issue in the same browser. 
What if my user insists on using Google Chrome, but his default browser is still set to Edge (or something else) due to system settings? (so the scenario: user opens Chrome, gets link in mail which auto opens in Edge when clicking the link and the localStorage that was previously set is unreacheable which causes the login to fail)

So long story short: How can I support the scenario that the login process via the link is completed in a different browser than the one that initiated the login sequence in the first place?

**Comment by damienbod** [comment]
closing this, please open if you still have an issue
